test/integration/targets/loop_control/inner.yml

```diff
@@ -0,0 +1,2 @@
+- debug:
+    msg: "{{ ansible_loop.index }} {{ ansible_loop.index0 }} {{ ansible_loop.revindex }} {{ ansible_loop.revindex0 }} {{ ansible_loop.first }} {{ ansible_loop.last }} {{ ansible_loop.length }} {{ ansible_loop.allitems|join(',') }}"
```

Comment:
i would change to asserts, but this seems fine also

Reply:
Probably only need to assert that `ansible_loop is defined`

Reply:
I added tests with `assert`. I tried to add as many tests I could think of without complicating things.